homeassistant/components/camera/webrtc.py

```diff
@@ -328,12 +329,20 @@ async def ws_get_client_config(
     )
 
 
+def _parse_webrtc_candidate_init(value: Any) -> RTCIceCandidateInit:
+    """Validate and parse a WebRTCCandidateInit dict."""
+    try:
+        return RTCIceCandidateInit.from_dict(value)
+    except (MissingField, ValueError) as ex:
+        raise vol.Invalid(str(ex)) from ex
+
+
 @websocket_api.websocket_command(
     {
         vol.Required("type"): "camera/webrtc/candidate",
         vol.Required("entity_id"): cv.entity_id,
         vol.Required("session_id"): str,
-        vol.Required("candidate"): str,
+        vol.Required("candidate"): _parse_webrtc_candidate_init,
```

Comment:
I's hard to see what the expected parameter argument is when there's no type annotation in `_parse_webrtc_candidate_init`. I'd at least add `dict[str, Any]` as type annotation.